source/getting-started-edge/index.rst

```diff
@@ -0,0 +1,245 @@
+
+Once you have PI status, you may apply for a new project with an initial
+allocation. A project application typically consists of a short description of
+your intended research and takes one business day to process. Once your project
+has been approved, you will be able to utilize the testbed sites.
+
+Step 3: Start using Chameleon!
+==============================
+
+Congratulations, you are now ready to launch your first container! Containers
+are a simple way to deploy applications. Learn more about what containers are 
+with `this guide from docker <https://www.docker.com/resources/what-container>`_.
+Follow these steps to launch a container and manage it.
+
+The CHI\@EDGE dashboard
+----------------------
+
+Chameleon edge resources are available through 
+`CHI@EDGE <https://chi.edge.chameleoncloud.org>`_. When you access this site, you are 
+first taken to a dashboard, which shows a summary of your project's current 
+resource usage. The dashboard looks  something like this:
+
+.. figure:: dashboard.png
+   :alt: The Chameleon Dashboard's resource usage summary
+   :figclass: screenshot
+
+   An overview of your project's current resource usage
+
+Setting up networking
+---------------------
+If you are planning to make your container accessible over the Internet with a
+Floating IP, your container will need to run a security group in order to expose
+the ports needed. Below is a brief introduction to security groups in Chameleon, 
+or you can find a more in depth explanation 
+`here <../technical/kvm.html#security-groups>`_.
+
+#. First, in the sidebar click *Network*, then click *Security Groups*. You can 
+   examine an existing group's rules by clicking *Manage Rules* next to it. If 
+   you already have a security group here that meets your needs, you can skip 
+   the next step. Otherwise, you will need to create a new security group.
+
+#. To create a new security group, click *+ Create Security Group*, enter a name 
+   for your new group in the wizard, and then click *Create Security Group*. 
+   You will be redirected to the screen to manage your new group's rules. Select 
+   *Add Rule* to open the add rule wizard. Under *Rule*, you can select from 
+   common rules, or if your needs are not met by one of these options, select 
+   one of the custom rules. For example, if your container runs a web
+   server, you may want to add the rules HTTP and HTTPS, allowing for traffic
+   on ports 80 and 443.
+
+  .. figure:: create_security_group.png
+     :alt: Create Security Group Wizard
+     :figclass: screenshot
+
+     Enter a name for the security group.
+
+Launching a container
+---------------------
+
+To start launching a container, follow the following steps:
+
+1. In the sidebar, click *Container*, then click *Containers*.
+
+2. Click on the *Create Container* button in the toolbar and the *Create
+   Container* wizard will load
+
+3. Type *my_first_container* for the container name. Then, enter the name of an 
```

Comment:
It may be good to be explicit that the container name can be anything. 

"Give your container a name. For example, since it's your first container, "my_first_container" might be a good name."